server/server.js

```diff
@@ -34,7 +33,31 @@ const pool = new pg.Pool({
   connectionString: "postgres://postgres:password@127.0.0.1:5432/pet_database"
 })
 
-// Express routes
+app.get("/", (req, res) => {
+  res.redirect("/pets")
+})
+
+app.get('/api/v1/pet_types', (req, res) => {
+  const petsTypeQuery = "SELECT * FROM pet_types"
+  pool.query(petsTypeQuery)
+    .then(result => {
+      res.send(result)
+    })
+    .catch(error => {
+      console.log(error)
+      res.sendStatus(500)
+    })
+})
+
+app.get('/api/v1/adoptable_pets', (req, res) => {
+  const type_id = req.query.type
+  let queryString = "SELECT * FROM adoptable_pets WHERE type_id = ($1)"
+  pool.query(queryString, [type_id])
+    .then(result => {
+      res.send(result)
```

Comment:
Both here and on line 44, I think you mean to use res.json, since these are api endpoints

Reply:
res.send and res.json are identical for our use case (sending an array): https://medium.com/gist-for-js/use-of-res-json-vs-res-send-vs-res-end-in-express-b50688c0cddf

Is that ok?

Reply:
As long as it works on the React end then that should be good.